java-examples/src/main/java/edu/practice/algos/sort/impl/MergeSorting.java

```diff
@@ -0,0 +1,57 @@
+package edu.practice.algos.sort.impl;
+
+public class MergeSorting {
```

Comment:
The name is confusing... what is the difference between MergeSort class and MergeSorting class?

Reply:
i will take care of this